### PR TITLE
Support building a new version after creating a service automatically

### PR DESCRIPTION
**Pull request by zoumo**
feat: Support building a new version after creating a service automatically
perf: add vcs ping function to accelerate service creation
refactor: change service and version id to mongodb objectId

**Changes**
   
- feat: Support building a new version after creating a service automatically
- perf: add vcs ping function to accelerate service creation
- refactor: change service and version id to mongodb objectId

**Reviewers**
- [ ] @superxi911  please review

**Checklist**

- [ ] Rebased/mergable
- [ ] Tests pass
- [ ] Issue/task done

### CODE REVIEW

File: event/manager.go [hunk 1]
```diff
@@ -168,13 +168,13 @@ func SaveEventToEtcd(event *api.Event) error {
 	if nil != err {
 		return err
 	}
-	return ec.Set(EventsUnfinished+"/"+string(event.EventID), string(bEvent))
+	return ec.Set(EventsUnfinished+string(event.EventID), string(bEvent))
```
Comment (gaocegege) [inline event/manager.go]: spaces between operators :smiley: 

```diff
- return ec.Set(EventsUnfinished+string(event.EventID), string(bEvent))
+ return ec.Set(EventsUnfinished + string(event.EventID), string(bEvent))
````

Comment (zoumo) [inline event/manager.go]: go tools auto format it for me, I can't change it
In this case, there are two params in function, each param should be tight I think😁

Comment (gaocegege) [inline event/manager.go]: got it :tada:

**Comment by superxi911** [comment]
LGTM

**Pull request merged**